templates/dashboard/product/modal_confirm_delete.html

```diff
@@ -1,37 +1,23 @@
+{% extends "dashboard/modal_confirm_delete.html" %}
+
 {% load i18n %}
 {% load prices_i18n %}
 {% load staticfiles %}
 {% load product_first_image from product_images %}
 
-<form action="{% url "dashboard:product-delete" pk=product.pk %}" method="post" novalidate>
-  {% csrf_token %}
-  <div class="modal-content">
-    <div class="row">
-      <div class="col s12">
-        <h5>
-          {% trans "Are you sure?" context "Dashboard confirmation modal" %}
-        </h5>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col s12 center-align">
-        <img src="{% product_first_image product size="255x255" method="crop" %}" 
-             srcset="{% product_first_image product size="255x255" method="crop" %} 1x, {% product_first_image product size="510x510" method="crop" %} 2x" 
-             alt="" class="responsive-img">
-      </div>
-    </div>
-    <p>
-      {% blocktrans trimmed context "Modal delete product text" %}
-        If you remove <strong>{{ product }}</strong> you will lose all data related to this product.
-      {% endblocktrans %}
-    </p>
-  </div>
-  <div class="modal-footer">
-    <button type="submit" class="modal-action btn-flat">
-      {% trans "Delete" context "Dashboard delete action" %}
-    </button>
-    <a href="#!" class="modal-action modal-close btn-flat">
-      {% trans "Cancel" context "Dashboard cancel action" %}
-    </a>
+{% block action_url %}{% url 'dashboard:product-delete' pk=product.pk %}{% endblock %}
+
+{% block content %}
+<div class="row">
+  <div class="col s12 center-align">
+    <img src="{% product_first_image product size="255x255" method="crop" %}" 
```

Comment:
Instead of product image we could have a content as in every other modal here ("Delete product x").